### PR TITLE
Setup the project

### PR DESCRIPTION
**Pull request by unitSphere**

### CODE REVIEW

File: .prettierrc.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "printWidth": 100,
+  "tabWidth": 2,
```
Comment (zilongqiu) [inline .prettierrc.json]: Seems we have 4 spaces currently in the .js files, right?

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
+node_modules/
```
Comment (zilongqiu) [inline .gitignore]: Can ignore the "usual" nodejs project files https://github.com/github/gitignore/blob/main/Node.gitignore

**Review by zilongqiu — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -1 +1,14 @@
...
+## Lalamove's Delivery SDK
+
+To install the dependencies run `npm install`
+
+To build the SDK run `npm run build`
+
+To test the SDK locally:
+
+1. Make sure you have built the SDK
+2. Run `mkdir testing` in project root
```
Comment (zilongqiu) [inline README.md:10]: Can we also create a Make command (ex: `make dev`) to execute these commands?

**Review by zilongqiu — APPROVED** [review]

File: index.ts [hunk 4]
```diff
@@ -0,0 +1,9 @@
+export function orderRequest(): string {
+    return "Made Order"
+}
+
+export function cancelOrder(id: number): string {
+    return "Cancelled Order with ID: " + id
```
Comment (ck-johnny) [inline index.ts]: The template string might be more clear. 
```suggestion
    return `Cancelled Order with ID:  ${id}`
```

File: .prettierrc.json (hunk 1 above)
Comment (unitSphere) [reply .prettierrc.json]: ops I didn't run it before pushing. Now I changed it to 4 and executed the styling

File: README.md (hunk 3 above)
Comment (unitSphere) [reply README.md:10]: of course

File: index.ts (hunk 4 above)
Comment (unitSphere) [reply index.ts]: Sure thing

File: .gitignore (hunk 2 above)
Comment (unitSphere) [reply .gitignore]: Wanted to add new things as we go. But sure thing, done.

File: Makefile [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+	npm run build
+
+test:
+	make build
+	mkdir testing && cd testing && npm init -y && npm install ..
+
+clean:
+	rm -R testing
+
+
```
Comment (zilongqiu) [inline Makefile:12]: A small detail but please remove this line break (only one line break at the end of the file)

**Pull request closed without merging**